packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -74,6 +75,12 @@ const inferCallWithChatControlOptions = (
  * @private
  */
 export const CallWithChatControlBar = (props: CallWithChatControlBarProps & ContainerRectProps): JSX.Element => {
+  useEffect(() => {
```

Comment:
consider call the hook named useRef + ref prop in react, it's a more react way, but gives you the same result

(if controlButton doesn't support that, use a basic react element instead,)

Reply:
Also, we don't need a data-ui-id

Reply:
Removed useEffect as I made the hidden button it's own reusable component as per your great suggestion.
https://github.com/Azure/communication-ui-library/pull/2601/files/eb637f038fb883e9edd0d10cb093a23944775bd3#r1051193481